functions/mockEndpoints.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+module.exports.upstreamPost = async (event, context) => {
```

Comment:
Really stupid simple endpoints, mainly here just to produce log messages so we can see things are working